unit/src/symbol-table/symbol_table_util.h

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************\
+
+ Module: Unit test utilities
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+/// \file
+/// Utility functions relating to  the sybmol table.
+
+#ifndef CPROVER_SRC_SYMBOL_TABLE_SYMBOL_TABLE_UTIL_H
+#define CPROVER_SRC_SYMBOL_TABLE_SYMBOL_TABLE_UTIL_H
+
+#include <util/symbol.h>
+
+class symbol_table_utilt
```

Comment:
You're quite right - @peterschrammel would this be an acceptable use of a `namespace`?